eval/evaluate-chat-prompt-flow.ipynb

```diff
@@ -11,22 +11,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "from promptflow import PFClient\n",
+    "\n",
     "pf_client = PFClient()\n",
     "\n",
     "from dotenv import load_dotenv\n",
     "\n",
-    "from pathlib import Path\n",
-    "load_dotenv(Path(\"../local.env\"))"
```

Comment:
Did you test this? I had issues when I removed the path. Did you?

Reply:
It worked fine for me, it found the root ".env". That's what it generally defaults to. Perhaps it didnt work before because it was local.env? ".env" is all it looks for, I believe.